packages/example-react/project/app/Request/components/index.js

```diff
@@ -1,5 +1,3 @@
-import Agreement from './Agreement';
-import Patient from './Patient';
-import Provider from './Provider';
-
-export { Agreement, Patient, Provider };
+export { Agreement } from './Agreement';
```

Comment:
At the end of the day, its all preference when it comes to readability vs. more lines of code. I do prefer the syntax you have in the PR but it fails to run due to you forgetting to export the components as default 
```javascript
export { default as Agreement } from './Agreement';
export { default as Patient } from './Patient';
export { default as Provider } from './Provider';

```

Reply:
Good catch! The change has been updated. 
Another alternative based on your feedback:
```
export { default } from './Agreement';
export { default } from './Patient';
export { default } from './Provider';
```

Reply:
Currently the pull request uses your suggestion:
```javascript
export { default as Agreement } from './Agreement';
export { default as Patient } from './Patient';
export { default as Provider } from './Provider';
```
And if this breaks convention in a terrible way or does not fit the company preference feel free to close the pr.

Reply:
> Good catch! The change has been updated.
> Another alternative based on your feedback:
> 
> ```js
> export { default } from './Agreement';
> export { default } from './Patient';
> export { default } from './Provider';
> ```

I don't think that actually works. The `babel-loader` will still think its exporting the default module for that file and not actually specific to the module import.

If you want feel free to finish this PR to all the others we are exporting. I prefer this syntax over the 'import/export` as I also follow this syntax in a lot of other apps well. Otherwise you can close this PR.

Reply:
@GoPro16, Correct that does not work. I should have tried it out before making the suggestion. Per your suggestion `export { default as Provider } from './Agreement';` does work. Also this pattern is present in Availity-react:
https://github.com/Availity/availity-react/blob/master/packages/hooks/index.js